photon-client/src/main/scala/com/linkedin/photon/ml/io/scopt/game/ScoptGameTrainingParametersParser.scala

```diff
@@ -138,9 +138,9 @@ object ScoptGameTrainingParametersParser extends ScoptGameParametersParser {
         usageText = "<value>"),
 
       // Hyper Parameter Tuner Name
-      ScoptParameter[HyperparameterTunerName, HyperparameterTunerName](
+      ScoptParameter[String, String](
         GameTrainingDriver.hyperParameterTunerName,
-        usageText = "<type>"),
+        usageText = "<class>"),
```

Comment:
Is the full class path provided here?

Reply:
Yes - I'll change the comment to be more specific.